File: algorithms/633. Sum of Square Numbers/solution.py
class Solution(object):
    def judgeSquareSum(self, c):
        """
        :type c: int
        :rtype: bool
        """
        for i in xrange(int(c ** 0.5) + 1):
            j = (c - i ** 2) ** 0.5
            if j == int(j):
                return True
        return False

File: algorithms/633. Sum of Square Numbers/solution.cc
class Solution {
public:
    bool judgeSquareSum(int c) {
        for ( long i = 0; i * i <= c; ++i ) {
            double j = sqrt(c - i * i);
            if ( j == int(j) )
                return true;
        }
        return false;
    }
};
